public/locations-list-map.js

```diff
@@ -476,19 +523,6 @@ function renderNextListPage() {
   const children = [];
 
   locationsListEntries.slice(renderLocation, renderLocation + 40).forEach(function (entry) {
-    // Add city/state headers
```

Comment:
Are we sure we want to get rid of city/state headers? It's weird for the list to jump from AK to AL with no break in between. May be a non-issue in the future once list respects map view, though I still think it will be weird when zoomed out to view country-wide or even multi-city results.

Reply:
Agreed! @tinabeans

Reply:
I'm thinking that once the list respects the map view it should be good enough for launch but ya I agree that city/state headers could be nice